cmd/ig/utils/flags.go

```diff
@@ -59,9 +59,7 @@ type CommonFlags struct {
 }
 
 func AddCommonFlags(command *cobra.Command, commonFlags *CommonFlags) {
-	command.PersistentPreRunE = func(cmd *cobra.Command, args []string) error {
-		command.Parent().PersistentPreRunE(command.Parent(), args)
-
+	command.PreRunE = func(cmd *cobra.Command, args []string) error {
```

Comment:
This commit is changing something that was added in the previous commit. Sorry, but I'm not sure I got what you are trying to do.

Reply:
Here, I set it to `PreRunE`, so the `bindViper` which is set as `PersistentPreRunE` can execute.
When the parent command and child command both have `PersistentPreRunE`, the child command's function would be called instead, so `bindViper` was not being called with `list-containers` command.